CHANGELOG.md

```diff
@@ -7,6 +7,18 @@ All notable changes to TTT2 will be documented here. Inspired by [keep a changel
 ### Added
 
 - Added ConVar to toggle double-click buying
+- Added a new vgui system
+  - the F1 menu is completely overhauled
+  - introduced new development interfaces to easily create menues and settings for addons
+- Introduced a global scale factor
+  - based on a 1080p resolution and automatically adjusted to the current screen resolution (i.e. a 4k display has a default scale factor of 2)
+  - used by the HUD
```

Comment:
this is way to detailed for a changelog entry imo

wouldn't something like the following suffice?
```suggestion
- Introduced a global scale factor based on screen resolution to scale HUD elements accordingly
```

Reply:
If you think yours is better, I'm fine with it